lang/java/reef-wake/wake/src/main/java/org/apache/reef/wake/remote/RemoteConfiguration.java

```diff
@@ -84,15 +100,15 @@
   /**
    * The number of tries.
    */
-  @NamedParameter(doc = "The number of tries.", default_value = "3")
+  @NamedParameter(doc = "The number of tries.", default_value = "" + REMOTE_CONNECTION_NUMBER_OF_RETRIES)
   public static final class NumberOfTries implements Name<Integer> {
     // Intentionally empty    
   }
 
   /**
    * The timeout of connection retrying.
    */
-  @NamedParameter(doc = "The timeout of connection retrying.", default_value = "10000")
+  @NamedParameter(doc = "The timeout of connection retrying.", default_value = "" + REMOTE_CONNECTION_RETRY_TIMEOUT)
```

Comment:
Based on your current numbers, it will be 20000/4 = 5000. It is shorter than previous one. Any reason to make it shorter?